core/os/os.cpp

```diff
@@ -73,6 +75,35 @@ void OS::initialize_logger() {
 	_set_logger(memnew(StdLogger));
 }
 
+void OS::initialize_physics() {
```

Comment:
I thought we discussed moving this code to main.cpp , similar to how AudioServer works, to a void a dependency from core to servers